assets/css/_search_page.scss

```diff
@@ -42,14 +42,6 @@ $search-page-search-panel-width: 22.0625rem;
     width: calc(100% - 2 * 1.5625rem);
     margin: 0 1.5625rem;
   }
-
-  &.visible {
-    left: 0;
-  }
-
-  &.hidden {
-    left: calc(-1 * #{$search-page-search-panel-width} + 1rem);
-  }
```

Comment:
I think you're right that this wasn't doing anything for the search page. It looks like I introduced it back in [this commit](https://github.com/mbta/skate/pull/1887/commits/9726fb39c5c73ed4aa751d878787e3e2de4bf3bf), I think in my zeal to make sure that all of the changes to the styling in the map page made it back to the search page I carried over some stuff about hiding the panel that isn't relevant.